src/EA.Weee.Core/DirectRegistrant/SellingTechniqueType.cs

```diff
@@ -10,6 +10,9 @@ public enum SellingTechniqueType
         IndirectSellingToEndUser = 1,
 
         [Display(Name = "Direct selling to end user (mail, order, internet etc) and Indirect selling (other)")]
-        Both = 2
+        Both = 2,
+
+        [Display(Name = "Online Marketplaces and Fulfilment Houses")]
```

Comment:
I thought about suggesting we give this new value 4 -- and then in future if we want an "all 3" option, it's trivial to add with value 6 -- but that won't quite work as the first value is 0. And it would be more work to get the Display text.